server/cmd/mmctl/commands/urls.go

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+const (
+	ConfigDocumentationUrl = "https://docs.mattermost.com/configure/environment-configuration-settings.html"
```

Comment:
@cwarnermm is there a PL we can use?

Reply:
Requested PL link from marketing:
- route: ``https://mattermost/com/pl/environment-configuration-settings``
- destination: ``https://docs.mattermost.com/configure/environment-configuration-settings.html``

Reply:
There is any news about it?

Reply:
Thanks for following up, @willypuzzle! Yes, this PL is now set up and available for use in-product.

Reply:
Oh, yes! But I don't know what a PL is and how to use it. Could you give me some explanation?

Reply:
please @hanzei help me :)

Reply:
@willypuzzle Sorry, I missed your ping. You can just use https://mattermost.com/pl/environment-configuration-settings in your code directly.